.ci/compute_projects.py

```diff
@@ -247,6 +255,15 @@ def _get_modified_projects(modified_files: list[str]) -> Set[str]:
         # capacity.
         if len(path_parts) > 3 and path_parts[:3] == ("llvm", "utils", "gn"):
             continue
+        # If the file is in the clang/lib/CIR directory, add the CIR project.
+        if len(path_parts) > 3 and (
+            path_parts[:3] == ("clang", "lib", "CIR")
+            or path_parts[:3] == ("clang", "test", "CIR")
+            or path_parts[:4] == ("clang", "include", "clang", "CIR")
+        ):
+            modified_projects.add("clang")
```

Comment:
If something in `clang/` is touched, `modified_projects` should already contain `clang`, no?

Reply:
If the only clang files modifed are CIR files, the `continue` statement here will keep clang from being added to the modified projects list. I could remove the `continue` and fall through, and that would add clang.

Reply:
OK